router/aspects/request_timeout_handler.go

```diff
@@ -3,11 +3,15 @@ package aspects
 import (
 	"github.com/julienschmidt/httprouter"
 	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/pbsmetrics"
 	"net/http"
 	"strconv"
+	"time"
 )
 
-func QueuedRequestTimeout(f httprouter.Handle, reqTimeoutHeaders config.RequestTimeoutHeaders) httprouter.Handle {
+const NanosecondsInSecond = 1000000000
```

Comment:
What about using `time.Second` instead? Then don't have to worry that you have the correct number of zeros. :)

Reply:
Sure, fixed!